build.py

```diff
@@ -9,17 +9,29 @@
 def create_parser():
     parser = argparse.ArgumentParser(description="Build PEP documents")
     # alternative builders:
-    parser.add_argument("-l", "--check-links", action="store_true")
-    parser.add_argument("-f", "--build-files", action="store_true")
-    parser.add_argument("-d", "--build-dirs", action="store_true")
+    parser.add_argument("-l", "--check-links", action="store_true",
+                        help='Checks validity of links within PEP sources. '
+                             'Cannot be used with "-f" or "-d".')
```

Comment:
If they cannot be used together, why not properly codify this as a [mutually exclusive argument group](https://docs.python.org/3/library/argparse.html#mutual-exclusion)? Then the arguments will be displayed appropriately automatically without the need to hardcode this message, it will fail early with an informative error message, and your code doesn't need to handle it yourself.

Reply:
> why not properly codify this as a mutually exclusive argument group

Laziness ;-) When originally putting it together it was a simpler thing, which grew larger. 

A